conversions.go

```diff
@@ -41,7 +41,9 @@ func convertInMessage(
 	inMsg *buffer.InMessage,
 	outMsg *buffer.OutMessage,
 	protocol fusekernel.Protocol) (o interface{}, err error) {
-	switch inMsg.Header().Opcode {
+	header := inMsg.Header()
+
+	switch header.Opcode {
```

Comment:
nit: use `switch header := inMsg.Header(); header.Opcode` to scope the `header` variable to the `switch` block only (smaller scopes are generally preferable)